apps/src/lib/ui/accounts/DeleteAccount.jsx

```diff
@@ -102,15 +104,32 @@ export default class DeleteAccount extends React.Component {
 
   state = DEFAULT_STATE;
 
-  toggleDialog = () => {
+  togglePersonalLoginDialog = () => {
     this.setState(state => {
       return {
         ...DEFAULT_STATE,
-        isDialogOpen: !state.isDialogOpen
+        isPersonalLoginDialogOpen: !state.isPersonalLoginDialogOpen
       };
     });
   };
 
+  toggleDeleteAccountDialog = () => {
+    this.setState(state => {
```

Comment:
TIL: I'd never seen the update function form of `setState` before.  Is it new?

Reply:
nice! i think it's relatively new. i first found it [in this article](https://medium.freecodecamp.org/functional-setstate-is-the-future-of-react-374f30401b6b) a while back, but i can't find in which version of react this was introduced